src/modules/ethereum/loanTransactions.js

```diff
@@ -269,9 +291,9 @@ function parseLoans(loansArray) {
             let isCollectable = false;
             let collateralStatus;
 
-            const state = parseInt(bn_state, 10);
+            const state = LOAN_STATES[parseInt(bn_state, 10)];
```

Comment:
this is not OK this way, it broke the loan status checks in other places (e.g. repay loan). 
I wouldn't use string for the status but a constant and roll this change everywhere else.
something like:
```js
const LoanState = Object.freeze( { OPEN: 0 , REPAID: 1, ...} );
```
![image](https://user-images.githubusercontent.com/7456451/42647773-d87515f8-85fc-11e8-8a53-ee8cc21a85c7.png)